docs/slack.rst

```diff
@@ -250,7 +250,7 @@ Self-promotion, surveys, and sales
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 You're welcome to share, link to, and mention your own work, including blog posts, social media, events, and paid products or services.
-That said, posts where you stand to benefit from others' money, personal information, or labor come with higher expectations.
+That said, posts where you stand to benefit from others' money, personal information, click-throughs, or labor come with higher expectations.
```

Comment:
I'm not 100% sure what this means. Maybe this?

```suggestion
That said, posts where you stand to benefit from others' money, personal information, attention, or labor come with higher expectations.
```

Reply:
click-throughs = ad revenue that is click/impression based. But I think 'attention' works there.